pkg/allowrules/allowrules.go

```diff
@@ -0,0 +1,73 @@
+package allowrules
+
+import (
+	"fmt"
+)
+
+type TransportProtoAllowRule struct {
+	serviceName string
```

Comment:
this structure should contain the protocol

Reply:
you can pull the rules for your particular protocol so theres no need to include it in the struct.
e.g.
```GetTCP()``` or ```GetUDP()```